examples/style_sheets/plot_bmh.py

```diff
@@ -1,21 +1,31 @@
 """
-This example demonstrates the "bmh" style, which is the design used in the
-Bayesian Methods for Hackers online book.
+=====================
+Bayesian Methods for Hackers style sheet
+=====================
+
```

Comment:
Yes, it does: our documentation build is else going to fail.

(To be precise, lines can be longer, but not shorter than the title)